project/dexter-monitor/public/service/serverStatusService.js

```diff
@@ -64,6 +64,16 @@ monitorApp.service('ServerStatusService', function($http, $log) {
         return inactiveServers.length;
     };
 
+    service.getActiveServerList = function() {
+        return $http.get("/api/v1/server")
+            .then((results) => {
+                return _.filter(results.data, {'active': true});
+            })
+            .catch((err) => {
+                $log.error(err);
```

Comment:
Added alert dialog at line 74.
`Failed to load active server list.`
`Please contact the system administrator`.